grails-app/controllers/org/pih/warehouse/order/OrderController.groovy

```diff
@@ -518,6 +524,69 @@ class OrderController {
         }
     }
 
+    def orderItemFormDialog = {
+        log.info "params " + params
+        OrderItem orderItem = OrderItem.get(params.id)
+        render(template: "orderItemFormDialog", model: [orderItem:orderItem])
+    }
+
+    def deleteOrderItem = {
+        OrderItem orderItem = OrderItem.get(params.id)
+        if (orderItem) {
+            Order order = orderItem.order
+            order.removeFromOrderItems(orderItem)
+            orderItem.delete()
+            order.save()
+            render (status: 201, text: "Successfully deleted order item")
```

Comment:
201 is reserved for 'Created'. You should use either 200, 202 or 204. In this case 200 is better suiting to the message.

Reply:
Thanks. I meant to put 204, but will use 200 since there's content.